sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 5.7.0-beta.2 (Unreleased)
+## 5.7.0-beta.2 (2022-02-08)
```

Comment:
Since we want to use 5.7.0 for the `skipParsingBodyAsJson` feature, this should be 5.8.0-beta.1